tests/test_database_matrix.py

```diff
@@ -382,8 +383,6 @@ async def test_default_update_same_key(patched_send, opsdroid_matrix):
     patched_send.return_value = nio.RoomGetStateEventResponse(
         {"twim": {"hello": "world"}}, "", "", ""
     )
-    patched_send.return_value.transport_response = AsyncMock()
-    patched_send.return_value.transport_response.status = 200
```

Comment:
Why are these being removed?

Reply:
They aren't needed any more because I changed the logic to not check the transport_response unless an error object is returned.